### PR TITLE
Add Question Answering Utility Using Transformer (#82)

### PR DESCRIPTION
**Pull request by antoprince001**
Flask web application to predict answers for questions from given text using BERT Question Answering model

BERT Model used : bert-large-uncased-whole-word-masking-finetuned-squad

Reference : https://towardsdatascience.com/question-answering-with-a-fine-tuned-bert-bc4dafd45626

image

### CODE REVIEW

**Comment by Anubhav-Goyal01** [comment]
Nice work @antoprince001
